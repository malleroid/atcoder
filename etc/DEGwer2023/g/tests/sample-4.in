4
0.800000 0.600000 0
1.000000 0.000000 1
-0.280000 -0.960000 1
0.000000 0.000000 0
