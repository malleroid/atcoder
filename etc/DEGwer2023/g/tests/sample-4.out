13.284042973728
